src/main/java/frc/robot/subsystems/base/climb/ClimberBase.java

```diff
@@ -0,0 +1,112 @@
+package frc.robot.subsystems.base.climb;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+/**
+ * @author Barel
+ * @version 1.0
+ * @since 2020-off
+ * <p>
+ * The class contains various methods that every {@code Climber} <b>should</b> implement.
+ * <p>
+ * The purpose of this class is to provide guidelines and restrict what important to us and what not.
+ */
+public abstract class ClimberBase extends SubsystemBase {
```

Comment:
I looked in the link, and I think it will be more appropriate to use abstract class instead of interface because, like in the answer, I want to define a behavior, it's not 

> a pattern.

Reply:
After a second thought, I think you're right.